documentation/_puppetserver_nav.html

```diff
@@ -6,58 +6,82 @@ <h3 id="puppetserver-{{server_version}}-docs">Puppet Server {{ server_version }}
     <ul>
       <li><a href="/puppetserver/{{ server_version }}/index.html">Index</a></li>
       <li><a href="/puppetserver/{{ server_version }}/services_master_puppetserver.html">About Puppet Server</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/release_notes.html">Release Notes</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/puppetserver_vs_passenger.html">Notable Differences vs. the Apache/Passenger Stack</a></li>
-      <li><a href="/puppetserver/{{ server_version }}/install_from_packages.html">Installation</a></li>
-      <li><strong>Configuring Puppet Server</strong>
-        <ul>
-          <li><a href="/puppetserver/{{ server_version }}/configuration.html">Puppet Server's Config Files</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_puppetserver.html">puppetserver.conf: Main Config File</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_auth.html">auth.conf: Access Control</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_webserver.html">webserver.conf: Jetty Web Server Config</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_web-routes.html">web-routes.conf: Mount Points for Component Services</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_global.html">global.conf: Trapperkeeper Settings</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_ca.html">ca.conf: CA Service Configuration</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_master.html">master.conf: Authorization by HTTP Header (deprecated)</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_product.html">product.conf: Configuring Product-level Interactions (optional)</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_logbackxml.html">logback.xml: Logging Level and Location</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_logging_advanced.html">Advanced Logging Configuration</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/bootstrap_upgrade_notes.html">Bootstrap Upgrade Notes</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/release_notes.html">Release notes</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/deprecated_features.html">Deprecated features</a></li>
```

Comment:
Does it make sense to call out deprecations this high up? I would sort of expect this more towards the bottom?

Reply:
It's normal across our docs to group deprecations alongside release notes. Deprecations are still changes users want to be able to find quickly so they know what's changed or expected to change.

Reply:
ahkay 👍